.travis.yml

```diff
@@ -11,11 +11,11 @@ env:
     - "JRUBY_OPTS=--dev"
 
 rvm:
-  - 2.1.10
-  - 2.2.9
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.3.8
+  - 2.4.5
+  - 2.5.5
+  - 2.6.2
+  - ruby-head
```

Comment:
If you are adding a potentially unstable build, can please mark failures as ignored? We don't want broken builds because we're testing on unstable software.

Reply:
Sorry for the late response...
I rethink about adding unstable build to CI, actually Adding unstable build seem makes no sense.
So I'll delete `ruby-head`.